tools/webdriver/webdriver/bidi/modules/script.py

```diff
@@ -0,0 +1,33 @@
+from typing import Any, Optional, Mapping, MutableMapping
+
+from ._module import BidiModule, command
+
+
+class Script(BidiModule):
+    @command
+    def evaluate(self,
+          expression: str,
+          context: Optional[str] = None,
+          realm: Optional[str] = None,
```

Comment:
I wonder if we should have a dedicated type for  `Target`, which we then most likely would have to share between different modules. 

```suggestion
          target: Target,
```

This would hardening the client that this argument is required and allows us to fail early without having to do the network roundtrip to the remote end.

Reply:
Done